pkg/kdump/config-client.js

```diff
@@ -273,7 +280,7 @@ export class ConfigFile {
                 if ("core_collector" in settings._internal)
                     settings._internal.core_collector.value = settings._internal.core_collector.value + " -c";
                 else
-                    settings._internal.core_collector = { value: "makedumpfile -c" };
+                    settings._internal.core_collector = { value: defaultCoreCollector };
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19434-20231010-112806-5f779cb5-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)